plugins/knife/knife-deployment-resize.rb

```diff
@@ -383,6 +415,7 @@ def run
           end
         end
 
+        update_ralf_hostname config[:environment], config[:cloud].to_sym
```

Comment:
Brackets (I know there are counter examples elsewhere in the file but that's not an excuse to make it worse :)).